packages/scxa-tsne-plot/src/TSnePlotView.js

```diff
@@ -114,12 +116,12 @@ class TSnePlotView extends React.Component {
     const plot = _find(plotTypeDropdown, (plot) => plot.plotType.toLowerCase() === selectedPlotType)
 
     const plotOptionsValues = plot.plotOptions.map((option) =>
-        ({value: Object.values(option)[0], label: Object.keys(option)[0]+`: `+Object.values(option)[0]}))
+      ({value: Object.values(option)[0], label: Object.keys(option)[0]+`: `+Object.values(option)[0]}))
 
     const plotTypesOptions = plotTypeDropdown.map((plot) => ({
-          value: plot.plotType.toLowerCase(),
-          label: plot.plotType
-        }))
+      value: plot.plotType.toLowerCase(),
+      label: plot.plotType
+    }))
 
     const getTooltipContent = async (cellId) => {
       const url = URI(`json/experiment/${this.props.experimentAccession}/cell/${cellId}/metadata`, atlasUrl).toString()
```

Comment:
The `accessKey` should be propagated to this request as well. Remember: if there’s an `experimentAccession` involved, then we should honour the `accessKey` field. Has the controller for this URL in the back end been modified?

Reply:
The controller already implements the logic correctly using accessKey so change in the frontend would suffice.